tasks.py

```diff
@@ -189,9 +189,9 @@ def _bump_version(version: str, bump: Optional[BumpType] = None) -> str:
     from packaging.version import Version
 
     v = Version(version)
-    if bump == BumpType.MAJOR:
+    if bump == BumpType.MAJOR.value:
         v = Version(f"{v.major + 1}.0.0")
-    elif bump == BumpType.MINOR:
+    elif bump == BumpType.MINOR.value:
```

Comment:
It wasn't possible to use `StrEnum` in this python module, since this module is used for installing all 3rd party deps, so it would be a catch-22